wiki/en/Running-a-Server.md

```diff
@@ -79,13 +79,14 @@ To run a Directory [read this guide](Custom-Directories)
 
 Most people run Jamulus as a "pure" Server on **hardware without audio** (e.g. on a 3rd party/cloud host) running Linux. The following steps assume you are familiar with the command line and Debian/Ubuntu or similar distribution using systemd. To run a server on Windows or on the desktop with a graphical user interface, [see this section](#servers-on-the-desktop).  
 
-If you want to run a Server on a Raspberry Pi, you will need to [compile from source](https://github.com/jamulussoftware/jamulus/blob/master/COMPILING.md). See also this [guide for Raspberry Pi](/kb/2020/03/28/Server-Rpi.html) maintained by Jamulus user fredsiva.
+If you want to run a Server on a Raspberry Pi (or a different armhf based device), you will need to download the armhf `.deb` files, not the default `amd64` ones you'd use on an Intel/AMD based machine.
+
 
 ### Installation
 
-1. Download the [latest headless .deb file]({{ site.download_root_link }}{{ site.download_file_names.deb-headless }})
+1. Download the [latest headless (amd64) .deb file]({{ site.download_root_link }}{{ site.download_file_names.deb-headless }}) or, if you use a Raspberry Pi etc. download the [latest armhf .deb file]({{ site.download_root_link }}{{ site.download_file_names.deb-headless-armhf }})
 1. Update apt to make sure you have a current list of standard packages: `sudo apt update`
-1. Install the Jamulus package: `sudo apt install ./{{ site.download_file_names.deb-headless }}`
+1. Install the Jamulus package: `sudo apt install ./{{ site.download_file_names.deb-headless }}` or for ARM: `sudo apt install ./{{ site.download_file_names.deb-headless-armhf }}`
```

Comment:
This wording needs to be made more consistent -- either shorten the first point to just "ARM" or use "Pi etc" here, perhaps?

Reply:
of for armhf sounded strange.

```suggestion
1. Install the Jamulus package: `sudo apt install ./{{ site.download_file_names.deb-headless }}` or for Pi (armhf): `sudo apt install ./{{ site.download_file_names.deb-headless-armhf }}`
```

Reply:
Maybe just "or for Pi (etc)", as `amd64` isn't mentioned for the first file?

Reply:
Done. See my latest commit